src/otx/algo/visual_prompting/segment_anything.py

```diff
@@ -49,80 +49,173 @@ class SegmentAnything(nn.Module):
 
```

Comment:
Can we divide nn.Module from OTXModels? 
We can use "segmentors" folder for VPT task
(detectors for OD, segmentors for IS, SS, VPT, classifiers for CLS)

Any other names are also considerable for folder, but for clearness and global structure, I propose to divide core models which is actually returned from _create_model().

Reply:
I agree with this and I have wanted to know where we can set the location of nn.Modules first because in your first proposal it seems not clear.